source/release-notes/backend.txt

```diff
@@ -13,6 +13,18 @@ Atlas App Services Changelog
 2024 Releases
 -------------
 
+.. _backend_20240724:
+
+24 July 2024 Release
+~~~~~~~~~~~~~~~~~~~~
+
+- GraphQL and Static Hosting Deprecation 
+
+  - Apps that use GraphQL or Static Hosting will now display banners in their App Services 
+    UI alerting customers of both the GraphQL and Static Hosting deprecations.
+
+  - The GraphQL template app (web.graphql.todo) is no longer available.
+
```

Comment:
Please add these other two items as well:
- Fixed bug where Triggers in a disabled state were not resuming correctly after being enabled
- Fixed bug where trigger deployments made in the Data Services UI were registering as Admin API rather than UI deployment